src/software/pipeline/main_featureMatching.cpp

```diff
@@ -100,6 +100,7 @@ int main(int argc, char **argv)
   int rangeSize = 0;
   std::string nearestMatchingMethod = "ANN_L2";
   std::string geometricEstimatorName = robustEstimation::ERobustEstimator_enumToString(robustEstimation::ERobustEstimator::ACRANSAC);
+  double geometricErrorMax = 0.0; //< the maximum reprojection error allowed for image matching with geometric validation
```

Comment:
I don't know which is the commenting convention, if there is one, but at some point we started to switch to
```c++
/// comment
double variable = 0.0;
```
like 
here https://github.com/alicevision/AliceVision/blob/develop/src/software/pipeline/main_cameraLocalization.cpp#L103
and many other places.
Anyway, nitpicking.